packages/docs/site/docs/developers/05-local-development/04-wp-playground-cli.md

```diff
@@ -117,8 +117,126 @@ The `server` command supports the following optional arguments:
+With the Playground CLI, you can use the `--help` flag to get a full list of available commands and arguments.
 
 ```bash
 npx @wp-playground/cli@latest --help
 ```
+
+## Programmatic Usage with JavaScript
+
+The Playground CLI can also be controlled programmatically from your JavaScript/TypeScript code using the `runCLI` function. This gives you direct access to all CLI functionalities within your development environment, allowing you to enhance and automate your workflows. A key benefit is the ability to create robust end-to-end tests. Let's cover the basics of using `runCLI`.
+
+### Running a WordPress instance with a specific version
+
+Using the `runCLI` function, you can specify options like the PHP and WordPress versions. In the example below, we request PHP 8.3, the latest version of WordPress, and to be automatically logged in. All supported arguments are defined in the `RunCLIArgs` type.
+
+```TypeScript
+import { runCLI, RunCLIArgs, RunCLIServer } from "@wp-playground/cli";
+
+let cliServer: RunCLIServer;
+
+cliServer = await runCLI({
+    command: 'server',
+    php: '8.3',
+    wp: 'latest',
+    login: true
+} as RunCLIArgs);
+```
+
+To execute the code above, the developer can set their preferred method. A simple way to execute this code is to save it as a `.ts` file and run it with a tool like `tsx`. For example: `tsx my-script.ts`
+
+### Setting a Blueprint
+
+You can provide a blueprint in two ways: either as an object literal directly passed to the `blueprint` property, or as a string containing the path to an external `.json` file.
+
+```TypeScript
+import { runCLI, RunCLIServer } from "@wp-playground/cli";
+
+let cliServer: RunCLIServer;
+
+cliServer = await runCLI({
+  command: 'server',
+  wp: 'latest',
+  blueprint: {
+    steps: [
+        {
+          "step": "setSiteOptions",
+          "options": {
+              "blogname": "Blueprint Title",
+              "blogdescription": "A great blog description"
+          }
+        }
+    ],
+  },
+});
+```
+
+For full type-safety when defining your blueprint object, you can import and use the `BlueprintDeclaration` type from the `@wp-playground/blueprints` package:
+
+```TypeScript
+import type { BlueprintDeclaration } from '@wp-playground/blueprints';
+
+const myBlueprint: BlueprintDeclaration = {
+  landingPage: "/wp-admin/",
+  steps: [
+    {
+      "step": "installTheme",
+      "themeData": {
+        "resource": "wordpress.org/themes",
+        "slug": "twentytwentyone"
+      },
+      "options": {
+        "activate": true
+      }
+    }
+  ]
+};
+```
+
+### Mounting a plugin programmatically
+
+Simply start with It is possible to mount local directories programmatically using `runCLI`. The options `mount` and `mount-before-install` are available. The `hostPath` property expects a path to a directory on your local machine. This path should be relative to where your script is being executed..
```

Comment:
One piece of feedback that keeps coming up is to avoid words like "simply" or "just" since it may not be simple for some folks. Let's stick to that.

Also, "Simply start with It is possible" seems like two different ways to start the same sentence :-)